src/__tests__/index.test.js

```diff
@@ -176,4 +176,10 @@ describe("PlaylistDetails integration tests", () => {
     screen.getAllByTestId("playlist-date").forEach((pl) => expect(pl).toBeInTheDocument());
     expect(screen.queryByRole("button", { name: "<< Back to show information" })).not.toBeInTheDocument();
   });
+
+  test("Tests that the see-full-schedule button correctly displays the schedule", () => {
+    fireEvent.click(screen.queryByRole("button", { name: "See Full Schedule" }));
+    expect(screen.queryByText("This is the schedule")).toBeInTheDocument();
```

Comment:
does the schedule still say "this is the schedule"?

Reply:
Yeah, so I made this branch before we had the schedule up and going, so its just that place holder text that isn't even a component. So this test will have to change to just see if <WeeklySchedule> is displayed once both things are merged.

Reply:
I think the easiest solution would be to merge this Pull Request, pull the changes into the branch with the actual schedule, and update the test there.